lib/file_previewer.dart

```diff
@@ -64,14 +63,12 @@ class FilePreview {
       );
       return image != null
           ? Image.memory(image.bytes)
-          : defaultImage ?? _defaultImage(filePath);
+          : defaultImage ?? _defaultImage;
     } catch (e) {
-      return defaultImage ?? _defaultImage(filePath);
+      return defaultImage ?? _defaultImage;
     }
   }
 
   /// In case a file preview cannot be properly rendered, show a placeholder image with the extension in the center
-  static Image _defaultImage(String filePath) {
-    return Image.network(placeholderImageUrl(extension(filePath)));
```

Comment:
Also remove placeholderImageUrl from the config.dart file. (Can delete entire file)